package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/spritezero",
-  "version": "7.0.0",
+  "version": "7.0.0-duelfuel.2",
```

Comment:
Just a reminder to update this before merging

Reply:
Done in 7d7caed